pkg/reconciler/route/resources/cluster_ingress_test.go

```diff
@@ -33,6 +35,14 @@ import (
 
 const ns = "test-ns"
 
+var ctx context.Context
```

Comment:
just use `context.Background()` directly in the test and initialize it with `cfg` right there. Local scoping is the best scoping :)

Reply:
I've turned it into a function rather than `func init()` for getting the context as it's used in multiple places.